cert-walletTests/Resources/sample_signed_cert-1.2.0.json

```diff
@@ -7,14 +7,14 @@
             "subtitle": "2016",
             "description": "Certificate description",
             "title": "Certificate title",
-            "image:certificate": "data:image/png;base64,",
+            "image": "data:image/png;base64,",
             "id": "https://www.theissuer.edu/criteria/2016/05/certificate-type.json",
             "issuer": {
                 "@type": "Issuer",
                 "url": "http://www.theissuer.edu",
                 "name": "Issuing Institution",
                 "id": "https://www.theissuer.edu/issuer/the-issuer.json",
-                "image:logo": "data:image/png;base64,",
+                "image": "data:image/png;base64,",

```

Comment:
actually...this passed due to issue #14. I intend to add more real samples after issuing some, and will update these samples before we address #14